services/hanke-service/src/main/resources/application.properties

```diff
@@ -11,3 +21,13 @@ management.endpoint.health.probes.enabled=liveness,readiness
 info.java-version = ${java.specification.version}
 info.java-vendor = ${java.specification.vendor}
 server.forward-headers-strategy=FRAMEWORK
+
+# LIQUIBASE
+# For Liquibase, until we can finalize the DataSource setup. Once that is done, Liquibase should
+# be able to use the (primary) data source of Spring Boot directly.
+# TODO: If testing will at some point need database "haitaton_test", that will need to handled
+# with e.g. Spring profiles
+spring.liquibase.url=jdbc:postgresql://${HAITATON_HOST:localhost}:${HAITATON_PORT:5432}/haitaton
```

Comment:
These will be removed in future and liquibase will use spring.datasource properties?

Reply:
Yes, now I noticed your comment above that.